02-express-tutorial/controllers/authorization.js

```diff
@@ -0,0 +1,31 @@
+
+const auth = (req, res) => {
```

Comment:
Don't forget to include 'next' as your third argument.  Your code throws an error when send a request to the /test route.